src/views/List.jsx

```diff
@@ -1,27 +1,11 @@
+import { Link } from 'react-router-dom';
 import { useEffect, useState } from 'react';
 import { ListItem } from '../components/ListItem';
-import { Link } from 'react-router-dom';
 
 export function List({ data, listPath }) {
 	const [search, setSearch] = useState('');
-	const [listName, setListName] = useState('');
-
-	const extractAfterSlash = (listPath) => {
-		const regex = /\/([^/]+)$/; // Matches the last / and captures everything after it
-		const match = listPath.match(regex);
-
-		if (match && match[1]) {
-			return match[1];
-		} else {
-			// If no match is found or the captured group is undefined, return an empty string or handle it as needed.
-			return '';
-		}
-	};
-
-	useEffect(() => {
-		if (!listPath) return;
-		setListName(extractAfterSlash(listPath));
-	}, [listPath]);
+	const [listName] = useState(listPath.split('/')[1]);
```

Comment:
I also realized that this doesn't even have to be state because it doesn't change, so could be `const listName = listPath.split('/')[1];`